cloud_images/centos7/install_prereqs.sh

```diff
@@ -35,10 +37,20 @@ ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdg /dcos/volume0
 ExecStart=/usr/local/sbin/dcos_vol_setup.sh /dev/xvdh /var/log
 
 [Install]
-WantedBy=local-fs.target
+RequiredBy=local-fs-pre.target
 EOF
 systemctl enable dcos_vol_setup
 
+echo ">>> Configure tmpfiles.d/chrony"
+cat > /etc/tmpfiles.d/chrony.conf <<EOF
+d /var/log/chrony 0755 chrony chrony -
```

Comment:
Right, I just checked, it's the journal directory that has setgid bit set. But that's taken care of by `systemd-tmpfiles`.